### PR TITLE
Refactor: Fixed docstring for flexLayout function

### PR DESCRIPTION
**Pull request by qwerty541**

### CODE REVIEW

File: src/common/utils.js [hunk 1]
```diff
@@ -173,8 +173,8 @@ const request = (data, headers) => {
  * @param {object} props Function properties.
  * @param {string[]} props.items Array of items to layout.
  * @param {number} props.gap Gap between items.
- * @param {number[]?=} props.sizes Array of sizes for each item.
- * @param {"column" | "row"?=} props.direction Direction to layout items.
+ * @param {number[]=} props.sizes Array of sizes for each item.
```
Comment (rickstaa) [inline src/common/utils.js:176]:
```suggestion
 * @param {"column" | "row"=} props.direction Direction to layout items.
```

File: src/common/utils.js [hunk 2]
```diff
@@ -173,8 +173,8 @@ const request = (data, headers) => {
...
+ * @param {"column" | "row"=} props.direction Direction to layout items.
```
Comment (rickstaa) [inline src/common/utils.js:177]:
```suggestion
 * @param {number[]=} props.sizes Array of sizes for each item.
```

Comment (rickstaa) [reply src/common/utils.js:177]: Maybe we can swap the order so that it is the same as in the function description?

**Review by rickstaa — CHANGES_REQUESTED** [review]

File: src/common/utils.js (hunk 2 above)
Comment (qwerty541) [reply src/common/utils.js:176]: Good suggestion, i have updated the code, check it please.

**Review by rickstaa — APPROVED** [review]

**Pull request merged**